packages/cli-plugin-metro/src/commands/start/watchMode.ts

```diff
@@ -37,38 +42,46 @@ function enableWatchMode(messageSocket: any, ctx: Config) {
     }
   });
 
-  process.stdin.on('keypress', (_key, data) => {
-    const {ctrl, name} = data;
-    if (ctrl === true) {
-      switch (name) {
-        case 'c':
-          process.exit();
-          break;
-        case 'z':
-          process.emit('SIGTSTP', 'SIGTSTP');
-          break;
-      }
-    } else if (name === 'r') {
-      messageSocket.broadcast('reload', null);
-      logger.info('Reloading app...');
-    } else if (name === 'd') {
-      messageSocket.broadcast('devMenu', null);
-      logger.info('Opening developer menu...');
-    } else if (name === 'i' || name === 'a') {
-      logger.info(`Opening the app on ${name === 'i' ? 'iOS' : 'Android'}...`);
-      const params =
-        name === 'i'
-          ? ctx.project.ios?.watchModeCommandParams
-          : ctx.project.android?.watchModeCommandParams;
-      execa('npx', [
-        'react-native',
-        name === 'i' ? 'run-ios' : 'run-android',
-        ...(params ?? []),
-      ]).stdout?.pipe(process.stdout);
-    } else {
-      console.log(_key);
+  const onPressAsync = async (key: string) => {
+    if (key === CTRL_C) {
+      process.exit();
     }
-  });
+
+    switch (key) {
+      case CTRL_Z:
+        process.emit('SIGTSTP', 'SIGTSTP');
+        break;
+      case 'r':
+        messageSocket.broadcast('reload', null);
+        logger.info('Reloading app...');
+        break;
+      case 'd':
+        messageSocket.broadcast('devMenu', null);
+        logger.info('Opening developer menu...');
```

Comment:
nit: ["Dev Menu"](https://reactnative.dev/docs/next/debugging#accessing-the-dev-menu)
```suggestion
        logger.info('Opening Dev Menu...');
```